src/webextension/list/containers/item-filter.css

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.filter {
```

Comment:
Pretty much all of the changes in this file belong in `src/webextension/widgets/filter-input.css`. This file should only be for component-specific overrides (and there actually shouldn't be any of those either).

Reply:
@jimporter, @linuxwolf helped me set up this css file in order to apply styles to the search bar in both instances (full view + doorhanger). I can move the styles to input.css, but how can I then target the search/filter bar specifically? I'd be happy to jump on a vidyo to discuss.

Reply:
@nickbrandt Just add a new class to the `<FilterInput/>` and write CSS as normal; there's a variable in `filter-input.js` called `finalClassName` that you could just prepend the style name too. It should be fairly straightforward to just copy the existing code.

This also applies to `<PasswordInput/>` in another PR.